source/intro.md

```diff
@@ -1117,7 +1116,7 @@ The suggestions that are shown after typing `pd.read` and pressing <kbd>Tab</kbd
 To get more info on the function you want to use,
 you can type out the full name
 and then hold <kbd>Shift</kbd> while pressing <kbd>Tab</kbd>
-to bring up a help dialogue including the same information as when using `help()`.
+to bring up a help dialogue including the same information as when using `?`.
 
```

Comment:
```suggestion
If you are working outside modern programming environments such as JupyterLab, VScode, and IPython,
you can access the help using the the `help()` function instead of `?`.

```

@trevorcampbell something like this? I don't like namedropping to new things to explain `help`, but I also don't want to get into what a REPL or "the basic python interpreter is".

Reply:
@joelostblom I was suggesting the other way around -- keep the original text on `help`, and then later in the JLab section introduce `?` as an alternative w/o much extra explanation. I agree we should avoid referencing technical stuff (REPLs etc) there, so we can just say it's specific to JLab and is nice because less typing.

I'll do a separate PR to show what I mean